pkgs/development/tools/misc/devspace/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "devspace";
+  version = "5.18.4";
+
+  src = fetchFromGitHub {
+    owner = "loft-sh";
+    repo = "devspace";
+    rev = "v${version}";
+    sha256 = "0s5117cgxgrxfki5drvg6d22dvrjffa03bi644zdl1p631r599r1";
+  };
+
+  vendorSha256 = null;
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${version}"
+  ];
+
+  doCheck = false;
```

Comment:
Is there a reason we disable the checks here?

Reply:
I had some trouble with the test framework that devspace uses, it involved deploying a working k8s cluster ([see their Github Actions](https://github.com/loft-sh/devspace/blob/master/.github/workflows/e2e-tests.yaml)) and I don't think it's worth it

Reply:
Ah, I see.
I would suggest adding a comment, so the next user isn't confused. 

Also, maybe consider passing a test to nixosTests in am VM, so we know, when something breaks. 
A k8s cluster could maybe be setup there. pgadmin for example tests that way (it expects a working postgres db, https://github.com/NixOS/nixpkgs/blob/2ea2f7b6d0cb7ce0712f2aa80303cda08deb0de2/nixos/tests/pgadmin4.nix).

Reply:
Okay, I will add a comment and start working on a `nixosTests` in a separated PR